Foundation/NSXMLDTDNode.swift

```diff
@@ -62,7 +62,7 @@ public class NSXMLDTDNode : NSXMLNode {
         }
     } //primitive
     
-    public override init(kind: NSXMLNodeKind, options: Int) {
+    public override init(kind: NSXMLNodeKind, options: NSXMLNodeOptionRawType) {
```

Comment:
I think because NSUInteger and NSInteger were preferred for APIs for future proofing the storage size for parameters.